deploy/deploy_conda.sh

```diff
@@ -1,23 +1,35 @@
 #!/usr/bin/env bash
 
+# quit script if any of the commands fails. Note that && commands should be in paranthesis for this to work.
+set -eo pipefail
+trap 'exit_status="$?" && echo Failed on line: $LINENO at command: $BASH_COMMAND && echo "exit status $exit_status" && exit' ERR
```

Comment:
I'm not amazing at bash but if this works as I assume it would it's perfect to solve [this issue](https://github.com/genomic-medicine-sweden/JASEN/issues/70)